channeldb/payment_control.go

```diff
@@ -183,6 +210,58 @@ func (p *PaymentControl) InitPayment(paymentHash lntypes.Hash,
 	return updateErr
 }
 
+// paymentIndexType indicates the type of index we have recorded in the payment
+// indexes bucket. This type indicates
+type paymentIndexType uint8
+
+// paymentIndexTypeHash is a payment index type which indicates that we have
+// created an index of payment sequence number to payment hash.
+const paymentIndexTypeHash paymentIndexType = iota
```

Comment:
nice addition 😎

we typically assign persisted enums to constants rather than using `iota`, otherwise reordering the definitions can break the mapping. not a blocker though, happy to change since i'll be extending this area in the near future